Gems/MotionMatching/Code/Source/FeatureVelocity.cpp

```diff
@@ -56,10 +69,10 @@ namespace EMotionFX::MotionMatching
         const Transform relativeToWorldTM = pose->GetWorldSpaceTransform(relativeToJointIndex);
 
         const AZ::Vector3 jointPosition = relativeToWorldTM.TransformPoint(jointModelTM.m_position);
-        const float scale = 0.15f;
-        const AZ::Vector3 velocityWorldSpace = relativeToWorldTM.TransformVector(velocity * scale);
+        const AZ::Vector3 velocityWorldSpace = relativeToWorldTM.TransformVector(velocity);
 
-        DebugDrawVelocity(debugDisplay, jointPosition, velocityWorldSpace, color);
+        const float scale = 0.1f;
```

Comment:
suggestion/question: Is it worth exposing this as an AZ_CVar if we'd ever want to change this to make it easier to see or reduce to hide it?

Reply:
Great suggestion, will get this in.